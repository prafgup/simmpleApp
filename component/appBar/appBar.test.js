import React from "react";
import {shallow} from "enzyme";

import AppBar from "./index";


describe("App Bar Profile",()=>{

    describe("App Bar",()=>{
        let component ;

        beforeEach(()=>{
            component = shallow(<AppBar/>);
        });

        it("Should match to snapshot",()=>{
            expect(component.instance()).toMatchSnapshot("Name Bio");
        });

        it("Should Render all Icons",()=>{
            expect(component.find("[data-test='appBar']").length).toBe(2);
        });
    })
});