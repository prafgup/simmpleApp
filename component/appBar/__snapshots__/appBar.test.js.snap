// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Bar Profile App Bar Should match to snapshot: Name Bio 1`] = `
AppBar {
  "context": Object {},
  "props": Object {},
  "refs": Object {},
  "setState": [Function],
  "state": null,
  "updater": Updater {
    "_callbacks": Array [],
    "_renderer": ReactShallowRenderer {
      "_context": Object {},
      "_didScheduleRenderPhaseUpdate": false,
      "_dispatcher": Object {
        "readContext": [Function],
        "useCallback": [Function],
        "useContext": [Function],
        "useDebugValue": [Function],
        "useDeferredValue": [Function],
        "useEffect": [Function],
        "useImperativeHandle": [Function],
        "useLayoutEffect": [Function],
        "useMemo": [Function],
        "useReducer": [Function],
        "useRef": [Function],
        "useResponder": [Function],
        "useState": [Function],
        "useTransition": [Function],
      },
      "_element": <AppBar />,
      "_firstWorkInProgressHook": null,
      "_forcedUpdate": false,
      "_instance": [Circular],
      "_isReRender": false,
      "_newState": null,
      "_numberOfReRenders": 0,
      "_renderPhaseUpdates": null,
      "_rendered": <View
        style={
          Object {
            "alignContent": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginLeft": 20,
            "marginRight": 20,
            "marginTop": 10,
          }
        }
      >
        <Icon
          allowFontScaling={false}
          color="orange"
          data-test="appBar"
          name="angle-left"
          size={30}
        />
        <Icon
          allowFontScaling={false}
          color="orange"
          data-test="appBar"
          name="bars"
          size={30}
        />
      </View>,
      "_rendering": false,
      "_updater": [Circular],
      "_workInProgressHook": null,
    },
  },
}
`;
